src/OSPSuite.Presentation/Presenters/Charts/ChartEditorPresenter.cs

```diff
@@ -474,6 +482,28 @@ public void SetCurveNameDefinition(Func<DataColumn, string> curveNameDefinition)
          _curveSettingsPresenter.CurveNameDefinition = curveNameDefinition;
       }
 
+      public void AddCurvesWithSameColorForColumn(IReadOnlyList<DataColumn> dataColumnList, CurveOptions defaultCurveOptions = null)
```

Comment:
That looks a lot like the code below. Can we harmonize?

Reply:
refactored it using a new private function